tendermint/src/validator.rs

```diff
@@ -30,11 +30,12 @@ impl TryFrom<RawValidatorSet> for Set {
             .into_iter()
             .map(TryInto::try_into)
             .collect();
-        Ok(Self::new(
-            unsorted_validators_result?,
-            value.proposer.map(TryInto::try_into).transpose()?,
-            Some(value.total_voting_power.try_into()?),
-        ))
+
+        Ok(Self {
+            validators: unsorted_validators_result?,
```

Comment:
This no longer sorts the validator set (which is done in `Set::new()`) - is that intentional?

Reply:
Oh good catch! Will fix this right now